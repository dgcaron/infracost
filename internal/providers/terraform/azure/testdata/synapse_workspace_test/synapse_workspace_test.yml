version: 0.1
resource_usage:
  azurerm_synapse_workspace.example:
    serverless_sql_pool_size_tb: 11