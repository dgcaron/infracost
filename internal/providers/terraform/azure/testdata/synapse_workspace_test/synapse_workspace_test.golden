
 Name                                                               Monthly Qty  Unit                      Monthly Cost 
                                                                                                                        
 azurerm_synapse_workspace.dataflows                                                                                    
 ├─ Serverless SQL pool size (first 10TB)                    Monthly cost depends on usage: $0 per TB                   
 ├─ Data flow - basic                                                         8  vCore-hour                       $2.14 
 ├─ Data flow - standard                                                     12  vCore-hour                       $4.14 
 ├─ Data pipeline azure hosted activity runs                 Monthly cost depends on usage: $1.00 per 1k activity runs  
 ├─ Data pipeline azure hosted data integration units        Monthly cost depends on usage: $0.25 per DIU-hour          
 ├─ Data pipeline azure hosted integration runtime           Monthly cost depends on usage: $0.005 per hours            
 ├─ Data pipeline azure hosted external integration runtime  Monthly cost depends on usage: $0.00025 per hours          
 ├─ Data pipeline self hosted activity runs                  Monthly cost depends on usage: $1.50 per 1k activity runs  
 ├─ Data pipeline self hosted data movement                  Monthly cost depends on usage: $0.10 per hours             
 ├─ Data pipeline self hosted integration runtime            Monthly cost depends on usage: $0.002 per hours            
 └─ Data pipeline self hosted external integration runtime   Monthly cost depends on usage: $0.0001 per hours           
                                                                                                                        
 azurerm_synapse_workspace.datapipelines                                                                                
 ├─ Serverless SQL pool size (first 10TB)                    Monthly cost depends on usage: $0 per TB                   
 ├─ Data flow - basic                                        Monthly cost depends on usage: $0.27 per vCore-hour        
 ├─ Data flow - standard                                     Monthly cost depends on usage: $0.35 per vCore-hour        
 ├─ Data pipeline azure hosted activity runs                                  3  1k activity runs                 $3.00 
 ├─ Data pipeline azure hosted data integration units                        31  DIU-hour                         $7.75 
 ├─ Data pipeline azure hosted integration runtime                          200  hours                            $1.00 
 ├─ Data pipeline azure hosted external integration runtime               4,000  hours                            $1.00 
 ├─ Data pipeline self hosted activity runs                                   1  1k activity runs                 $1.50 
 ├─ Data pipeline self hosted data movement                                  10  hours                            $1.00 
 ├─ Data pipeline self hosted integration runtime                           200  hours                            $0.40 
 └─ Data pipeline self hosted external integration runtime                4,000  hours                            $0.40 
                                                                                                                        
 azurerm_synapse_workspace.general                                                                                      
 ├─ Serverless SQL pool size (first 10TB)                                    10  TB                               $0.00 
 ├─ Serverless SQL pool size (over 10TB)                                      1  TB                               $5.00 
 ├─ Data flow - basic                                        Monthly cost depends on usage: $0.27 per vCore-hour        
 ├─ Data flow - standard                                     Monthly cost depends on usage: $0.35 per vCore-hour        
 ├─ Data pipeline azure hosted activity runs                 Monthly cost depends on usage: $1.00 per 1k activity runs  
 ├─ Data pipeline azure hosted data integration units        Monthly cost depends on usage: $0.25 per DIU-hour          
 ├─ Data pipeline azure hosted integration runtime           Monthly cost depends on usage: $0.005 per hours            
 ├─ Data pipeline azure hosted external integration runtime  Monthly cost depends on usage: $0.00025 per hours          
 ├─ Data pipeline self hosted activity runs                  Monthly cost depends on usage: $1.50 per 1k activity runs  
 ├─ Data pipeline self hosted data movement                  Monthly cost depends on usage: $0.10 per hours             
 ├─ Data pipeline self hosted integration runtime            Monthly cost depends on usage: $0.002 per hours            
 └─ Data pipeline self hosted external integration runtime   Monthly cost depends on usage: $0.0001 per hours           
                                                                                                                        
 azurerm_synapse_workspace.vnet                                                                                         
 ├─ Serverless SQL pool size (first 10TB)                                    10  TB                               $0.00 
 ├─ Data flow - basic                                        Monthly cost depends on usage: $0.27 per vCore-hour        
 ├─ Data flow - standard                                     Monthly cost depends on usage: $0.35 per vCore-hour        
 ├─ Data pipeline azure hosted activity runs                 Monthly cost depends on usage: $1.00 per 1k activity runs  
 ├─ Data pipeline azure hosted data integration units        Monthly cost depends on usage: $0.25 per DIU-hour          
 ├─ Data pipeline azure hosted integration runtime           Monthly cost depends on usage: $1.00 per hours             
 ├─ Data pipeline azure hosted external integration runtime  Monthly cost depends on usage: $1.00 per hours             
 ├─ Data pipeline self hosted activity runs                  Monthly cost depends on usage: $1.50 per 1k activity runs  
 ├─ Data pipeline self hosted data movement                  Monthly cost depends on usage: $0.10 per hours             
 ├─ Data pipeline self hosted integration runtime            Monthly cost depends on usage: $0.002 per hours            
 └─ Data pipeline self hosted external integration runtime   Monthly cost depends on usage: $0.0001 per hours           
                                                                                                                        
 OVERALL TOTAL                                                                                                   $27.33 
----------------------------------
To estimate usage-based resources use --usage-file, see https://infracost.io/usage-file

1 resource type wasn't estimated as it's not supported yet.
Please watch/star https://github.com/infracost/infracost as new resources are added regularly.
1 x azurerm_storage_account